### ISSUE TITLE
[Feature Request] Firmware Updates for Modules

### ISSUE THREAD

**Issue by Veldkornet** [athombv/com.fibaro#214]
Fibaro have stated that they are willing to work with other gateway manufacturers in order to provide firmware updates to the modules. 

Can we please seriously look into this? It’s something that everyone with Fibaro devices want. 

See here for more information: https://forum.fibaro.com/topic/21711-fibaro-device-firmware-list/?page=2

**Comment by caseda** [comment]
I give the chance of this "business agreement" very low, why? Fibaro has *no benefits* in giving athom the firmware.

As they also state "if both manufacturers agree on sharing firmware updates", athom has no firmware they can share with fibaro.

But athom can try, just don't get your hopes up high, we have tried multiple times (not just me) to get fibaro to give any indication of giving firmware, and that always ended up in _no_

**Comment by Veldkornet** [comment]
Agreed that there’s no garuntees. But they could try, negotiate, etc. 

If customers can’t update a device that has crap firmware, eventually people will lose faith in that brand, and switch to something else. 
So really it’s in Fibaro’s interest to do this. 
Personally, if I were looking at buying a device from the various brands, if I know that Homey could update the firmware of a device I’d more likely buy that device over another brand since any issues could be fixed.

**Comment by caseda** [comment]
If you really think that the small amount of people (0.1% or so) that would buy another brand instead of fibaro because of bad firmware (which is not possible for some modules they produce) and don't want to go to a shop to update will hurt them, then they would have and given firmware data long ago, like the 6-7 years OTA is available, is a long time already

**Comment by Veldkornet** [comment]
Well if every had your attitude, nothing would ever change.

**Comment by caseda** [comment]
After almost 2 years of trying, you wouldn't? :wink:

**Comment by lbink** [comment]
Yes please! 👍 

Or alternatively a PC App for updates only that can use a Z.Wave dongle.

**Comment by Veldkornet** [comment]
Looks like Aeotec are keen to do this -> [Aeotec firmware updates come to HomeSeer](url
https://aeotec.com/home-automation-news/2138-aeotec-firmware-upgrade-homeseer.html)

**Comment by kaohlive** [comment]
+1 here. I would not even mind buying a Fibaro gateway device for it if there is a way for homey to stay in control For my Hue I use the API app to control my devices from Homey. But for the Fibaro app its direct or nothing. And once we are direct we can not use the official update anymore without a lot of manual labor (I do not do smart home to add a lot of maintenance tasks ;)). So I would love to be able to update my Fibaro devices from the Homey app.

And I would assume with the release of the Google Home thing and other big players taking control of bridges it would become a requirement for a company like Fibaro, they sell units as being compatible with these devices, so they should be able to update the firmware also. And if Fibaro allows google to update they should allow Homey to update also.
